awx/main/models/oauth.py

```diff
@@ -57,17 +57,3 @@ def is_valid(self, scopes=None):
             self.save(update_fields=['last_used'])
         return valid
 
-
-class OAuth2RefreshToken(AbstractRefreshToken):
-
-    class Meta:
-        app_label = 'main'
-        verbose_name = _('refresh token')
-        
-    application = models.ForeignKey(
```

Comment:
Out of curiosity, what did we originally have this here for?

Reply:
This was originally to solve an application FK problem I ran into with creating Personal Access Tokens... I figured a much better way to fix that, but had already overriden the RefreshToken.  I left it in at the time because I thought it might be useful for customizing these models in the future.  

tl;dr: we don't need it.